src/js/themes/base.js

```diff
@@ -623,6 +623,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'status-disabled',
           opacity: 'medium',
         },
+        border: {
+          // color: undefined,
+        },
+        label: {
+          // color: undefined,
+        },
```

Comment:
Can we please add those types to base.d.ts to enhance typescript support?

Reply:
And since those objects are empty I'm thinking they should be commented out.

Reply:
Will do.

Reply:
Done.